bin/prettier.js

```diff
@@ -146,6 +147,8 @@ function getTrailingComma() {
 }
 
 const options = {
+  astTransform: argv["ast-transform"] &&
+    require(path.resolve(process.cwd(), argv["ast-transform"])),
```

Comment:
thanks, fixed in https://github.com/prettier/prettier/pull/1710/commits/4620e9b9d76f42f0c7498529a0111bdadc646020